snuba/datasets/plans/single_storage.py

```diff
@@ -119,7 +121,9 @@ def __init__(
 
     def build_plan(self, request: Request) -> ClickhouseQueryPlan:
         storage = self.__selector.select_storage(request.query, request.settings)
-        clickhouse_query = QueryTranslator().translate(request.query)
+        clickhouse_query = QueryTranslator(TranslationMappers()).translate(
```

Comment:
this will pick between multiple TranslationMappers depending on the selected storage.